docs/notebooks/wasserstein_barycenters_gmms.ipynb

```diff
@@ -0,0 +1,497 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    ys, bs, n_log_prob = jax.vmap(generate_random_gmm, in_axes=(None, None, None, 0, 0, None, None, 0), out_axes=(0, 0, 0))(n_components, dim, b_cost, cs, ss, obs, None, seeds)
</pre><p>To highlight the versatility of the algorithm, I think we should experiment with various weights a as well.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/89/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>ok</p>